yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -353,6 +355,18 @@ func (yb *TargetYugabyteDB) CreateVoyagerSchema() error {
 			rows_imported BIGINT,
 			PRIMARY KEY (data_file_name, batch_number, schema_name, table_name)
 		);`, BATCH_METADATA_TABLE_NAME),
+		fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
+			channel_no INT PRIMARY KEY,
+			last_applied_vsn BIGINT);`, EVENT_CHANNELS_METADATA_TABLE_NAME),
+		fmt.Sprintf(`CREATE OR REPLACE FUNCTION %s(channel_no_val INT, last_applied_vsn_val BIGINT) RETURNS VOID AS $$ 
```

Comment:
@amit-yb Considered some alternatives here:
- init table - but that means adding another method to the targetDB interface to init with NUM_CHANNELS, so I decided to skip it for now. 
- insert on conflict update. This works but the majority of the operations on this are updates, so it's suboptimal to attempt an insert every time.